src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge31.java

```diff
@@ -0,0 +1,90 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+
+import java.nio.charset.StandardCharsets;
+
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.ChallengeTechnology;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.List;
+import java.util.UUID;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.owasp.wrongsecrets.RuntimeEnvironment.Environment.DOCKER;
+
+@Component
+@Order(31)
+public class Challenge31 extends Challenge {
+
+    public Challenge31(ScoreCard scoreCard) {
+        super(scoreCard);
+    }
+
+    static String getanswer() {
```

Comment:
```suggestion
    private String getanswer() {
```

Reply:
Sorry @commjoen, Since I had used the same method in Challenge31Test.java to call out spoiler. I couldn't commit directly i had to do some modifications in the code.

Reply:
i've made the changes requested, 
Could you please review it again